sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestInheritedAnnotations.java

```diff
@@ -85,7 +85,9 @@ public void testCrud() {
     }
 
     @RegisterConstructorMapper(Character.class)
-    public interface CharacterDao extends CrudDao<Character, Integer> {}
```

Comment:
This modifies the existing test. Can we add a second test that explicitly looks for the non-direct supertypes?